packages/frontend/src/model/settings.ts

```diff
@@ -44,22 +43,71 @@ export const AllSettingsQuery = selector({
   },
 });
 
-export const AllSettings = atom<Setting[]>({
+export const SingleSetting = atomFamily<SettingDto | undefined, string>({
+  key: 'SingleSetting',
+  default: undefined,
+});
+
+export const AllSettingIds = atom<string[] | undefined>({
+  key: 'SettingIdList',
+  default: undefined,
```

Comment:
Should AllSettingIds default to an empty array instead of undefined? Then we don't need to allow the 'undefined' type.

Reply:
actually nevermind i see that's how we're doing it with periods. Let's just keep the same implementations.